cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -125,6 +128,27 @@ def service(appliance, ansible_catalog_item):
         service_.delete()
 
 
+@pytest.fixture
```

Comment:
I know its deprecated, but we use `yield_fixture` decorator everywhere else. I'd like to keep it consistent until we decide to drop it and just use fixture everywhere.